### ISSUE TITLE
how to record only internal audio 

### ISSUE THREAD

**Issue by Krg72** [chinmoyp/screenrecorder#13]
is there a way to record screen with audio source as system audio only
and not mic

**Comment by WorldWonderful10** [comment]
Yes, since Android 10. [See](https://medium.com/@juliozynger/media-projection-and-audio-capture-1ca72e271e9c).
